From 6249deb04ab4a10d3fcad48074bec0e3e6be2a49 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Johan=20Bergstro=CC=88m?= <johan@pageload.io>
Date: Fri, 1 Mar 2019 19:59:36 -0300
Subject: [PATCH] tscore/ink_hrtime: add missing cstdint import

On alpine linux 3.9 (and edge) the compilation was failing
because of a missing import.

Refs: https://github.com/apache/trafficserver/issues/5103
---
 include/tscore/ink_hrtime.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/tscore/ink_hrtime.h b/include/tscore/ink_hrtime.h
index a1405a98c7..f321476913 100644
--- a/include/tscore/ink_hrtime.h
+++ b/include/tscore/ink_hrtime.h
@@ -33,6 +33,7 @@
 #include "tscore/ink_config.h"
 #include "tscore/ink_assert.h"
 #include <ctime>
+#include <cstdint>
 #include <sys/time.h>
 #include <cstdlib>
 typedef int64_t ink_hrtime;
